tx-pool/src/service.rs

```diff
@@ -516,8 +565,10 @@ impl TxPoolServiceBuilder {
             Arc::clone(&last_txs_updated_at),
         );
 
+        let txs = tx_pool.load_from_file()?;
```

Comment:
I think the start should not block by load file failed. if this failed under some circumstances，just clean the file，do not fail to start.